pwiz_tools/Skyline/Model/AuditLog/AuditLogEntry.cs

```diff
@@ -660,7 +660,11 @@ public AuditLogEntry ChangeUndoAction(Action<AuditLogEntry> undoAction)
 
         public AuditLogEntry ChangeExtraInfo(string extraInfo)
         {
-            return ChangeProp(ImClone(this), im => im.ExtraInfo = extraInfo);
+            return ChangeProp(ImClone(this), im =>
+            {
```

Comment:
Should the setter for ExtraInfo do this? Are their other cases where it makes sense to set ExtraInfo without doing this?